static/elements/chromedash-featurelist.js

```diff
@@ -369,16 +351,24 @@ class ChromedashFeaturelist extends LitElement {
   render() {
     return html`
       <link rel="stylesheet" href="/static/css/elements/chromedash-featurelist.css">
-
-      ${this.filtered.map((feature) => html`
-        <div class="item">
-          <div ?hidden="${this._computeMilestoneHidden(feature, this.features, this.filtered)}"
-               class="milestone-marker">${this._computeMilestoneString(feature.browsers.chrome.status.milestone_str)}</div>
-          <chromedash-feature id="id-${feature.id}" tabindex="0"
-              @feature-toggled="${this._onFeatureToggled}"
-              .feature="${feature}" ?whitelisted="${this.whitelisted}"></chromedash-feature>
-        </div>
-        `)}
+      <style>
+        .item {width: 100%}
+      </style>
+
+      <lit-virtualizer
+        .scrollTarget=${window}
+        .items=${this.filtered}
+        @rangechange=${this._onScrollList}
+        .renderItem=${(feature) => html`
+          <div class="item">
+            <div ?hidden="${this._computeMilestoneHidden.call(this, feature, this.features, this.filtered)}"
+                 class="milestone-marker">${this._computeMilestoneString.call(this, feature.browsers.chrome.status.milestone_str)}</div>
```

Comment:
Why `this._computeMilestoneString.call(this, ...`? That's the same as `this._computeMilestoneString(...`

Reply:
The context for function calls inside `renderItem` is `lit-virtualizer` instead of this component (`chromedash-featurelist`). So I have to bind/call them with this.

Not sure if it's designed behavior of lit-virtualizer.

Reply:
Oh, after some more experiments, it turns out only the event handler need this binding (and changed to a static arrow function).

Reply:
would it be possible to bind the function in the constructor?

```
constructor() {
  super();
  this._computeMilestoneString = this._computeMilestoneString.bind(this);
}
```

If that doesn't work, it might be good to add a comment explaining why we used `call()`.